verilog/analysis/verilog_linter.cc

```diff
@@ -147,23 +147,27 @@ absl::Status VerilogLinter::Configure(
     syntax_tree_linter_.AddRule(std::move(rule));
   }
 
+  absl::Status rc = absl::OkStatus();
   if (!configuration.external_waivers.empty()) {
```

Comment:
If you pass `absl::SkipEmpty()` to `StrSplit` you can remove this `if` check and rely only on the `for` loop.

Reply:
Sure, will do!